RORD eraser: scaffold (#1)
finegrain-ai/LBM: contrib/eraser/README.md

## LBM Eraser

> Reproduce Eraser result from LBM paper on SD1.5.

Reviewer: ```suggestion
**Goal**: reproduce the object-removal result from the LBM paper (Section 4.1) using SD1.5.
```

Non-official implementation.

### Dataset

> Prepare a group [webdataset](https://github.com/webdataset/webdataset) shards organized like 

Reviewer: ```suggestion
Prepare a group of [webdataset](https://github.com/webdataset/webdataset) shards, organized like:
```

```
<id0>.after.jpg
<id0>.before.jpg
<id0>.mask.png
<id1>.after.jpg
<id1>.before.jpg
<id1>.mask.png
...

> ```

Reviewer: ```suggestion
```

Where "before" is the image containing the object to remove, "mask" is the (fine) segmentation mask of the object in the before image, and "after" is the image without that object (ground truth).

```

### Train

To train the model on a single GPU machine, you can use the following command.

> Works on a single RTX 3090  

Reviewer: ```suggestion
Works on a single RTX 3090:
```

```bash
# Mock the SLURM env variables
export SLURM_JOB_ID=0000 
export SLURM_PROCID=0 
export SLURM_ARRAY_TASK_ID=0
export SLURM_NPROCS=1
export SLURM_NODEID=0
export SLURM_NNODES=1
export SLURM_LOCALID=0 

# neptune mode, 

> export NEPTUNE_MODE=async # use 'offline' to disable neptune logging

Reviewer: ```suggestion
# Neptune (https://neptune.ai) settings. Use 'offline' mode to disable logging.
export NEPTUNE_MODE=async
```
